_docs/sysadmin/installation/system_customization.md

```diff
@@ -298,4 +298,15 @@ Additional links will appear to the right of the copyright notice and credit lin
 
 
 
+## Customizing the login screen
+
+You can customize the login screen with markdown. By default, `# Login` is rendered which you can replace or add onto.
+
+1. Create an empty file `login.md` and place it in Submitty's system configuration folder.
+  * The configuration folder is typically `/usr/local/submitty/config`.
+2. The `login.md` file must be readable by the `submitty_php` user.
+  * e.g. Set ownership of the file to `root:submitty_php` and permissions to `RW-R-----`.
```

Comment:
Let's reword the permissions as `640 (RW-R-----)` so that it's easier to copy/paste (though we could probably drop the text description).

Reply:
I copied that from one of the above comments so I changed it on mine and where I copied it from.